projects/ngx-translate/core/src/lib/translate.directive.ts

```diff
@@ -72,28 +72,21 @@ export class TranslateDirective implements AfterViewChecked, OnDestroy {
         if (forceUpdate) {
           node.lastKey = null;
         }
-        if(isDefined(node.lookupKey)) {
-          key = node.lookupKey;
```

Comment:
I believe that the lookup key will prevent the translation of an updated interpolated bound template variable.